### DISCUSSION TITLE
You can animate entities (items, switches, goals, teleporters, billboards)

### DISCUSSION THREAD

**Discussion by parasti** [Neverball/neverball#361] [Show and tell]
I really gave it away in the title, but check this out. If you can compile from source, you can use these features in your maps right now. I don't have time for Windows builds (found the time after all: https://github.com/Neverball/neverball/releases/tag/1.7.0-alpha.1) Neverba. https://play.neverball.org supports these as well, but you won't be able to tell because there are no maps that use these features yet.

https://github.com/Neverball/neverball/assets/179160/a379de0f-29e8-4815-963d-afc1deb53a46

The Neverball gamepack bundled with TrenchBroom didn't really work for me, so I made a new one. You can also find that in the source in a folder called TrenchBroom. I used TrenchBroom because NetRadiant constantly crashed for me while I was developing this. By the way, [TrenchBroom](https://trenchbroom.github.io/) is awesome and you should check it out, it will probably skyrocket your mapping.

**Comment by fwp** [comment]
Wow, that's really something! 😮

**Reply by parasti** [reply]
Just my humble contribution to Nevermania, hint hint. Happy to see you're still around, honestly.

**Reply by fwp** [reply]
Yeah, I'm still around. Sadly, life hasn't really been compatible with Neverball-related activities lately.

As for Nevermania, I'd still like to add new levels. The more immediate focus, however, would be to continue fixing/improving the existing ones. So I think it's more likely we'll first see other mappers making use of the new functionality. After all, they created quite a lot of cool levels utilizing geometry rotation once that was available.

I have yet to take a close look at your enhancements, but they already seem great.
